ios/ReactNativeBrownfield.swift

```diff
@@ -32,6 +32,13 @@ class ReactNativeBrownfieldDelegate: RCTDefaultReactNativeFactoryDelegate {
   private var onBundleLoaded: (() -> Void)?
   private var delegate = ReactNativeBrownfieldDelegate()
   
+  private func checkFactoryInitialized(launchOptions: [AnyHashable: Any]? = nil) {
+    if reactNativeFactory == nil {
+      delegate.dependencyProvider = RCTAppDependencyProvider()
+      self.reactNativeFactory = RCTReactNativeFactory(delegate: delegate)
+    }
+  }
```

Comment:
Can we create w getter on the factory that initizalizes it if it's nil? It should remove the need to call this function